src/ProductConstructionService/ProductConstructionService.Api/Pages/DependencyFlow/Incoming.cshtml.cs

```diff
@@ -16,7 +16,7 @@ namespace ProductConstructionService.Api.Pages.DependencyFlow;
 
 public class IncomingModel : PageModel
 {
-    private static readonly Regex _repoParser = new(@"https?://(www\.)?github.com/(?<owner>[A-Za-z0-9-_\.]+)/(?<repo>[A-Za-z0-9-_\.]+)");
+    private static readonly Regex _repoParser = new Regex(@"https?://(www\.)?github.com/(?<owner>[A-Za-z0-9-_\.]+)/(?<repo>[A-Za-z0-9-_\.]+)");
```

Comment:
does this not work for some reason?

Reply:
I copied this, we can revert the change

```suggestion
    private static readonly Regex _repoParser = new(@"https?://(www\.)?github.com/(?<owner>[A-Za-z0-9-_\.]+)/(?<repo>[A-Za-z0-9-_\.]+)");
```